### PR TITLE
Added ports webpage help

### PR DESCRIPTION
**Pull request by pablomendezroyo**
Added webpage for ports mapping when autodiagnos for UpNp fails

### CODE REVIEW

File: packages/admin-ui/src/pages/support/formaters/autoDiagnoseTexts.ts [hunk 1]
```diff
@@ -87,15 +87,16 @@ export function openPorts({
...
   return {
     ok: !alertToOpenPorts,
     msg: alertToOpenPorts
       ? "Ports have to be opened and there is no UPnP device available"
       : "No ports have to be opened OR the router has UPnP enabled",
     solutions: [
       "If you are capable of opening ports manually, please ignore this error",
-      "Your router may have UPnP but it is not turned on yet. Please research if your specific router has UPnP and turn it on"
+      "Your router may have UPnP but it is not turned on yet. Please research if your specific router has UPnP and turn it on",
+      `Check your ports: https://check-host.net/check-udp?host=${internalIp}%3A1194`
```
Comment (dapplion) [inline packages/admin-ui/src/pages/support/formaters/autoDiagnoseTexts.ts:99]: @eduadiez Shouldn't this use the public IP instead of the internal IP?

Comment (pablomendezroyo) [reply packages/admin-ui/src/pages/support/formaters/autoDiagnoseTexts.ts:99]: right! my bad

File: packages/admin-ui/src/pages/support/formaters/autoDiagnoseTexts.ts [hunk 2]
```diff
@@ -87,15 +87,16 @@ export function openPorts({
...
   return {
     ok: !alertToOpenPorts,
     msg: alertToOpenPorts
       ? "Ports have to be opened and there is no UPnP device available"
       : "No ports have to be opened OR the router has UPnP enabled",
     solutions: [
       "If you are capable of opening ports manually, please ignore this error",
-      "Your router may have UPnP but it is not turned on yet. Please research if your specific router has UPnP and turn it on"
+      "Your router may have UPnP but it is not turned on yet. Please research if your specific router has UPnP and turn it on",
+      `Check your ports: https://check-host.net/check-udp?host=${publicIp}%3A1194`
```
Comment (dapplion) [inline packages/admin-ui/src/pages/support/formaters/autoDiagnoseTexts.ts:99]: Be a bit more helpful in the message. Explain what this page does, why is it useful for this problem and even consider adding a second link with a forum guide explaining how to do it.

**Comment by dapplion** [comment]
Closing for https://github.com/dappnode/DNP_DAPPMANAGER/pull/613

**Pull request closed without merging**